README.md

```diff
@@ -63,6 +63,8 @@ curl -H "Authorization: Bearer xxxx" \
 
 > Please get a token from Kubernetes cluster, and replace `xxxx` with it.
 
+If  you want to see ks-devops postman api collection , please visit **[ks-devops postman](https://www.postman.com/ks-devops/workspace/kubesphere-devops)**
```

Comment:
There is an extra space here.

`api` -> `API`

Reply:
thank you , it's  fixed  now